taraxa/state/dpos/precompiled/dpos_contract.go

```diff
@@ -876,6 +911,24 @@ func (self *Contract) state_put(key *common.Hash, state *State) {
 	}
 }
 
+func (self *Contract) check_validator_owner(owner, validator *common.Address) bool {
+	key := stor_k_1(field_validator2owner, validator[:])
+	var saved_addr common.Address
+	self.storage.Get(key, func(bytes []byte) {
+		saved_addr = common.BytesToAddress(bytes)
+	})
+	return *owner == saved_addr
+}
+
+func (self *Contract) set_validator_owner(owner, validator *common.Address) {
```

Comment:
this function should not exist. The whole point of separate Validators struct is to encapsulate details like this into the class implemenation so anyone who will work on the contract dont need to think about it. Delete this fucntion and put:

- self.storage.Put(key, nil) into DeleteValidator
- self.storage.Put(key, owner.Bytes()) into CreateValidator